azure-pipelines.yml

```diff
@@ -178,10 +178,10 @@ jobs:
       maxParallel: 6
       matrix:
         node-10-a:
-          node_version: ^10.13.0
+          node_version: ^12.22.12
           part: a
         node-10-b:
-          node_version: ^10.13.0
+          node_version: ^12.22.12
           part: b
         node-12-a:
           node_version: ^18.0.0
```

Comment:
I noticed these fields seem misaligned (e.g. `node-12-a` has `node_version` 18). This happened in the prior upgrade: https://github.com/webpack/webpack/pull/18083